modules/appeals_api/app/workers/appeals_api/daily_stuck_records_report.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'sidekiq/monitored_worker'
+
+class AppealsApi::DailyStuckRecordsReport
+  include Sidekiq::Worker
+  include Sidekiq::MonitoredWorker
+
+  # Only retry for ~8 hours since the job runs daily
+  sidekiq_options retry: 11, unique_for: 8.hours
+
+  def perform
+    params = find_stuck_record_data(AppealsApi::HigherLevelReview) + \
+             find_stuck_record_data(AppealsApi::NoticeOfDisagreement) + \
+             find_stuck_record_data(AppealsApi::SupplementalClaim)
+
+    AppealsApi::Slack::Messager.new(params, notification_type: :stuck_record).notify!
+  end
+
+  def enabled?
+    Settings.vsp_environment == 'production' &&
+      Settings.modules_appeals_api.reports.daily_stuck_records.enabled
```

Comment:
Is this right? It seemed like we'd only want these reports for production, and I left this setting disabled in `settings.yml`, since it seemed like all the other reports were disabled there as well, but lmk if that's wrong